src/index.html

```diff
@@ -9,6 +9,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#home" class="logo">
+        <img
+          class="logo-img"
+          src="./images/logo.png"
+          alt="asd">
+      </a>
+      <nav class="nav__list">
+        <ul>
+          <li>
+            <a href="#" class="is-active">Apple</a>
+          </li>
+          <li>
+            <a href="#">Samsung</a>
+          </li>
```

Comment:
```suggestion
          <li>
            <a href="#" class="is-active">Apple</a>
          </li
          >
          <li>
            <a href="#">Samsung</a>
          </li>
```

Add blank line between multiple sibling blocks of html